README.md

```diff
@@ -38,14 +38,14 @@ Please note that you will certainly need to provide your GitHub credentials with
 A lot of our dependencies are coming from GitHub and this reaches the max limit of 50 API calls from anonymous users.
 
 ```
-    $ php composer.phar create-project --prefer-dist akeneo/pim-community-standard ./pim-project "2.1.*@stable"
+    $ php composer.phar create-project --prefer-dist akeneo/pim-community-standard ./pim-project "2.2.*@stable"
```

Comment:
2.3-dev@dev ?

Reply:
Normally should be done in the 2.3 release no ? But ok I'll do it

Reply:
As the master branch has the alias 2.3, I think 2.3-dev@dev can work. It will have to be changed for 2.3.*@stable when 2.3 is released.